4-css-grid/styles/styles.css

```diff
@@ -0,0 +1,37 @@
+.A {
```

Comment:
use lowercase classnames. 
```suggestion
.a {
```